spec/util/bundler_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+RSpec.describe PDK::Util::Bundler do
+  before(:each) do
+    allow(PDK::CLI::Exec).to receive(:execute).and_return({exit_code: 0})
+  end
+
+  describe '.ensure_bundle!' do
+    context 'when there is no Gemfile' do
+      before(:each) do
+        allow(File).to receive(:file?).with(/Gemfile$/).and_return(false)
+      end
+
+      it 'does nothing' do
```

Comment:
Well, technically a puppet module doesn't have to have a Gemfile right? Theoretically somebody could still use `pdk` to run lint or something on such a module. Did we decide to ignore any use cases that did not involve a module that was generated with `pdk new`?